### PR TITLE
Rework style of dev command showing extensions details

### PR DESCRIPTION
**Pull request by stloyd**
All extensions:
```shell
stloyd@MacBook-Pro-2 static-php-cli % bin/spc dev:extensions 
     _        _   _                 _           
 ___| |_ __ _| |_(_) ___      _ __ | |__  _ __  
/ __| __/ _` | __| |/ __|____| '_ \| '_ \| '_ \ 
\__ \ || (_| | |_| | (_|_____| |_) | | | | |_) |
|___/\__\__,_|\__|_|\___|    | .__/|_| |_| .__/   v2.0.0
                             |_|         |_|    
Extensions:
 ------------ ----------------------------------------------------------------- ------------------------------------- -------------- --------------- ----------- 
  Extension    lib-suggests                                                      lib-suggests                          ext-depends    ext-suggests    unix-only  
 ------------ ----------------------------------------------------------------- ------------------------------------- -------------- --------------- ----------- 
  apcu                                                                                                                                                false      
  bcmath                                                                                                                                              false      
  bz2          bzip2                                                                                                                                  false      
  calendar                                                                                                                                            false      
  ctype                                                                                                                                               false      
  curl         zlib, openssl, curl                                                                                                                    false      
  dba                                                                                                                                                 false      
  dom          libiconv, zlib, libxml2                                                                                                                false      
  event        zlib, openssl, libevent                                                                                 openssl        sockets         false      
  exif                                                                                                                                                false      
  ffi          libffi                                                                                                                                 false      
  fileinfo                                                                                                                                            false      
  filter                                                                                                                                              false      
  ftp                                                                            openssl                                                              false      
  gd           zlib, libpng                                                      libavif, libwebp, libjpeg, freetype   zlib                           false      
  gettext                                                                                                                                             false      
  glfw         glfw                                                                                                                                   false      
  gmp          gmp                                                                                                                                    false      
  iconv                                                                                                                                               false      
  imagick      zlib, libpng, libjpeg, bzip2, libwebp, freetype, imagemagick                                                                           false      
  imap                                                                           kerberos                                                             false      
  inotify                                                                                                                                             false      
  intl         icu                                                                                                                                    false      
  ldap                                                                                                                                                false      
  mbregex      onig                                                                                                    mbstring                       false      
  mbstring     onig                                                                                                                                   false      
  memcache     zlib                                                                                                    session                        false      
  memcached    zlib, libmemcached                                                                                      session,zlib                   false      
  mongodb                                                                        icu, openssl, zstd, zlib                                             false      
  mysqli       zlib                                                                                                    mysqlnd                        false      
  mysqlnd      zlib                                                                                                                                   false      
  opcache                                                                                                                                             false      
  openssl      zlib, openssl                                                                                                                          false      
  pcntl                                                                                                                                               true       
  pdo                                                                                                                                                 false      
  pdo_mysql    zlib                                                                                                    pdo,mysqlnd                    false      
  pdo_pgsql    libiconv, zlib, libxml2, openssl, ncurses, readline, postgresql                                         pdo,pgsql                      false      
  pdo_sqlite   sqlite                                                                                                  pdo,sqlite3                    false      
  pgsql        libiconv, zlib, libxml2, openssl, ncurses, readline, postgresql                                                                        false      
  phar         zlib                                                                                                    zlib                           false      
  posix                                                                                                                                               true       
  protobuf                                                                                                                                            false      
  pspell                                                                                                                                              false      
  readline     ncurses, readline                                                                                                                      false      
  redis                                                                                                                               session         false      
  session                                                                                                                                             false      
  shmop                                                                                                                                               false      
  simplexml    libiconv, zlib, libxml2                                                                                                                false      
  snappy       zlib, snappy                                                                                                                           false      
  snmp                                                                                                                                                false      
  soap         libiconv, zlib, libxml2                                                                                                                false      
  sockets                                                                                                                                             false      
  sodium       libsodium                                                                                                                              false      
  sqlite3      sqlite                                                                                                                                 false      
  ssh2         zlib, openssl, libssh2                                                                                                                 false      
  swoole       zlib, openssl                                                                                           openssl        curl, pgsql     true       
  swow                                                                           openssl, curl                                        openssl, curl   false      
  sysvmsg                                                                                                                                             true       
  sysvsem                                                                                                                                             true       
  sysvshm                                                                                                                                             true       
  tidy                                                                                                                                                false      
  tokenizer                                                                                                                                           false      
  xlswriter    zlib, libzip                                                                                            zlib,zip                       false      
  xml          libiconv, zlib, libxml2                                                                                                                false      
  xmlreader    libiconv, zlib, libxml2                                                                                                                false      
  xmlwriter    libiconv, zlib, libxml2                                                                                                                false      
  xsl          libiconv, zlib, libxml2, libxslt                                                                        xml,dom                        false      
  yaml         libyaml                                                                                                                                false      
  zip          zlib, libzip                                                                                                                           false      
  zlib         zlib                                                                                                                                   false      
  zstd         zstd                                                                                                                                   false      
 ------------ ----------------------------------------------------------------- ------------------------------------- -------------- --------------- ----------- 
```

Limited extensions:
```shell
stloyd@MacBook-Pro-2 static-php-cli % bin/spc dev:extensions snappy zstd
     _        _   _                 _           
 ___| |_ __ _| |_(_) ___      _ __ | |__  _ __  
/ __| __/ _` | __| |/ __|____| '_ \| '_ \| '_ \ 
\__ \ || (_| | |_| | (_|_____| |_) | | | | |_) |
|___/\__\__,_|\__|_|\___|    | .__/|_| |_| .__/   v2.0.0
                             |_|         |_|    
Available extensions:
 ----------- -------------- -------------- ------------- -------------- ----------- 
  Extension   lib-suggests   lib-suggests   ext-depends   ext-suggests   unix-only  
 ----------- -------------- -------------- ------------- -------------- ----------- 
  snappy      zlib, snappy                                               false      
  zstd        zstd                                                       false      
 ----------- -------------- -------------- ------------- -------------- ----------- 


```

Unknown:
```shell
stloyd@MacBook-Pro-2 static-php-cli % bin/spc dev:extensions invalid    
     _        _   _                 _           
 ___| |_ __ _| |_(_) ___      _ __ | |__  _ __  
/ __| __/ _` | __| |/ __|____| '_ \| '_ \| '_ \ 
\__ \ || (_| | |_| | (_|_____| |_) | | | | |_) |
|___/\__\__,_|\__|_|\___|    | .__/|_| |_| .__/   v2.0.0
                             |_|         |_|    
Available extensions:

                                                                                                                        
 [WARNING] Unknown extension selected: invalid                                                                          
                                                                                                                        

```

### CODE REVIEW

File: src/SPC/command/dev/AllExtCommand.php [hunk 1]
```diff
@@ -6,23 +6,61 @@
...
+                implode(', ', $libraries),
+                implode(', ', $lib_suggests),
+                implode(',', $not_included),
+                implode(', ', $ext_suggests),
+                Config::getExt($extension, 'unix-only', false) ? 'true' : 'false',
+            ];
+        }
+
+        $style->table(
+            ['Extension', 'lib-suggests', 'lib-suggests', 'ext-depends', 'ext-suggests', 'unix-only'],
```
Comment (crazywhalecc) [inline src/SPC/command/dev/AllExtCommand.php:60]: `'lib-suggests', 'lib-suggests'` ===> `'lib-depends', 'lib-suggests'`

Comment (stloyd) [reply src/SPC/command/dev/AllExtCommand.php]: Missed, but fixed ;)

**Review by crazywhalecc — APPROVED** [review]

**Comment by crazywhalecc** [comment]
The unix-only column seems to be of little use at the moment. It could be changed to `supported-os` later when more systems are supported.

**Comment by crazywhalecc** [comment]
I happened to use the `dev:extensions` command today. Would it be better to create a unified format?

> use commas to separate multiple extension names

Because the current build command, Action build, download, and optional library addition are all separated by commas, using spaces to separate them here may cause confusion.

**Comment by stloyd** [comment]
Space is used cause it's array of arguments, but it can be changed of course to use one argument and check for commas

**Pull request merged**